README.md

```diff
@@ -1,4 +1,47 @@
-# managed-velero-operator
+# Managed Velero Operator
+
+## Summary
+
+The Managed Velero Operator is used for backups in OpenShift Dedicated v4. It is based on an open source project [Velero]( https://velero.io) that uses a controller model where it monitors custom resources and takes actions based on their states. The Managed Velero Operator dynamically creates and configures prerequisites for Velero and it then deploys the Velero software on the OpenShift Dedicated cluster. Velero backs up the Kubernetes object store: all the deployments, pods, config maps, secrets, etc. In addition to Kubernetes configuration pieces, Velero also handles backing up and any persistent volumes that are attached to the cluster. For OpenShift Dedicated v4, these are AWS EBS volumes.
```

Comment:
It's not just EBS volumes, as we also support GCP. We can make this more generic and say something like "we snapshot the cloud provider's persistent storage volumes"